src/domain/resource/card/__tests__/__snapshots__/ResourceCard.test.js.snap

```diff
@@ -105,6 +106,43 @@ exports[`ResourceCard renders correctly 1`] = `
       icon="test-file-stub"
       text="null"
     />
+    <span
+      className="app-resourceCardInfoCell"
+    >
+      <span
+        className="app-ResourceCardInfoCell__loginMethods"
+      >
+        <img
+          alt="Suomi.fi"
+          className="app-resourceCardInfoCell__icon"
+          key="Suomi.fi"
+          src="test-file-stub"
+          title="Suomi.fi"
+        />
+        <img
+          alt="Google"
+          className="app-resourceCardInfoCell__icon"
+          key="Google"
+          src="test-file-stub"
+          title="Google"
+        />
+        <img
+          alt="Facebook"
+          className="app-resourceCardInfoCell__icon"
+          key="Facebook"
+          src="test-file-stub"
+          title="Facebook"
+        />
+        <img
+          alt="Yle Tunnus"
+          className="app-resourceCardInfoCell__icon"
+          key="Yle Tunnus"
+          src="test-file-stub"
+          title="Yle Tunnus"
+        />
+      </span>
+      ResourceCard.requiredLoginMethods
```

Comment:
Dose this really use the translated string?

Reply:
I think it uses the current active language to find the translation keys.